packages/task/src/browser/task-service.ts

```diff
@@ -398,6 +398,17 @@ export class TaskService implements TaskConfigurationClient {
         }
     }
 
+    async runTaskByLabel(taskLabel: string): Promise<Boolean> {
+        const tasks: TaskConfiguration[] = await this.getConfiguredTasks();
```

Comment:
Should it be applied to detected tasks as well or `getConfiguredTasks` return all tasks, i.e. configured by a user and detected by extensions? cc @theia-ide/task-extension

Reply:
From what I understand getConfiguredTasks only returns user configured tasks (defined in tasks.json) so if we wanted them to run detected tasks it should be 'getTask'.

Reply:
I've changed this to getTasks as that seems like what would be needed but should their be different priority's? As for example what if someone registers a keybind for a label that matches both a detected task and a user configured one, currently it would be whatever is first in the array I think it should be user configured first then detected?

Reply:
I think it is fine to just use `getTasks`, since -IIUC- a configured task is a complete override by the user of a given detected task (cc @elaihau?)

Reply:
I think it is fine to just use `getTasks`, since -IIUC- a configured task is a complete override by the user of a given detected task (cc @elaihau?)

Reply:
*test*